docs/fastdds/xml_configuration/making_xml_profiles.rst

```diff
@@ -32,9 +40,11 @@ The following sections will show implementation examples for each of these profi
 Loading and applying profiles
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In case the user defines the |Entity| profiles via XML files, it is required to load these
-XML files using the |load_XML_profiles_file| public member function before creating any |Entity|.
-Moreover, |create_participant_with_profile|, |create_publisher_with_profile|, and |create_subscriber_with_profile|
+In case the user defines the |Entity-api| profiles via XML files, it is required to load these
+XML files using the |DomainParticipantFactory::load_XML_profiles_file-api| public member function before creating any
+entity.
+Moreover, |DomainParticipantFactory::create_participant_with_profile-api|,
+|DomainParticipant::create_publisher_with_profile-api|, and |DomainParticipant::create_subscriber_with_profile-api|
```

Comment:
[create_publisher_with_profile] and [create_subscriber_with_profile] are not clickable